documentation/API_documentation/customer-insights-API-Readiness-Checklist.md

```diff
@@ -0,0 +1,28 @@
+# API Readiness Checklist
+
+Checklist for Customer Insights v0.1.0-rc.1 in r1.1
+
+| Nr | API release assets  | alpha | release-candidate |  initial<br>public | stable<br> public | Status | Comments |
```

Comment:
As you have copied the new instructions in and they are referring to the column "Reference Information":

```suggestion
| Nr | API release assets  | alpha | release-candidate |  initial<br>public | stable<br> public | Status | Reference Information |
```